api/environments/identities/traits/views.py

```diff
@@ -71,11 +72,22 @@ def get_identity_from_request(self):
         """
         return Identity.objects.get(pk=self.kwargs["identity_pk"])
 
+    def _send_identity_update_message(self):
+        identity = self.get_identity_from_request()
+        send_identity_update_message.delay(
+            args=(
+                identity.environment.api_key,
+                identity.identifier,
+            )
+        )
+
     def perform_create(self, serializer):
         serializer.save(identity=self.get_identity_from_request())
+        self._send_identity_update_message()
```

Comment:
Couldn't we just pull the identity into a variable and do this logic in place? 

```python
def perform_create(self, serializer):
    identity = self.get_identity_from_request()
    serializer.save(identity=identity)
    send_identity_update_message.delay(...)
```

it feels off that we're calling `self.get_idnetity_from_request()` twice in quick succession.